src/api/migrations/20190620192200-create-user.js

```diff
@@ -0,0 +1,70 @@
+export default {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Users', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: Sequelize.INTEGER
+    },
```

Comment:
I hope you will rebase after they merge the migrations PR @MCFrank16

Reply:
yeah  definitely